### PR TITLE
[Fix](Outfile) Fixed the problem that the concurrent Outfile wrote multiple Success files

### PR DESCRIPTION
**Pull request by BePPPower**
## Proposed changes

Issue Number: close #xxx

**Problem:**
When we enable concurrent `Outfile` and specify the `success_file_name`, a SUCCESS file is written for each BE instance, which is not what we expected

**solution:**
Therefore, we added a new RPC request that when the Outfile was completed, the FE sent an RPC to the Be request to write one Success file.

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by BePPPower** [comment]
run buildall

**Comment by BePPPower** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/qe/StmtExecutor.java [hunk 1]
```diff
@@ -1883,6 +1903,75 @@ private void sendResult(boolean isOutfileQuery, boolean isSendFields, Queriable
         }
     }
 
+    private void outfileWriteSuccess(OutFileClause outFileClause) throws Exception {
+        if (!Strings.isNullOrEmpty(outFileClause.getSuccessFileName())) {
```
Comment (morningman) [inline fe/fe-core/src/main/java/org/apache/doris/qe/StmtExecutor.java:1907]: Already check it

File: fe/fe-core/src/main/java/org/apache/doris/qe/StmtExecutor.java [hunk 2]
```diff
@@ -1883,6 +1903,75 @@ private void sendResult(boolean isOutfileQuery, boolean isSendFields, Queriable
...
+            // 1. set TResultFileSinkOptions
+            TResultFileSinkOptions sinkOptions = new TResultFileSinkOptions(outFileClause.getFilePath(),
```
Comment (morningman) [inline fe/fe-core/src/main/java/org/apache/doris/qe/StmtExecutor.java:1909]: Why not using `outfileClause.toSinkOptions()`?

**Comment by BePPPower** [comment]
run buildall

**Comment by BePPPower** [comment]
run buildall

**Comment by morningman** [comment]
run buildall

**Comment by BePPPower** [comment]
run buildall

**Comment by BePPPower** [comment]
run buildall

**Review by morningman — APPROVED** [review]
LGTM

**Review by wuwenchi — APPROVED** [review]

**Pull request merged**